src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -662,10 +663,12 @@ def _create_command_arguments(self):
         return response.commandid
 
     def _get_static_info(self):
-        request = DataModelProtoModule.GetStaticInfoRequest()
-        request.rules = self.rules
-        response = self.service.get_static_info(request)
-        return response.info
+        if self.rules not in PyCommand._stored_static_info.keys():
```

Comment:
@prmukherj please could you add a comment here to the effect that we know that the system has already obtained these same infos remotely in another context and we need to coordinate the code so that the global infos are commonly available in a ll contexts (without additional remote calls).

Reply:
@seanpearsonuk , Done. Thank you